src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -102,9 +102,11 @@ export function fetchTagDistribution(
   key: string,
   query: EventQuery
 ): Promise<Tag> {
-  const urlParams = pick(query, Object.values(URL_PARAM));
+  // const urlParams = pick(query, Object.values(URL_PARAM));
```

Comment:
Could delete this.

Reply:
Reverting this and putting it in a separate PR.